terraform/workspaces/paragon/modules.tf

```diff
@@ -14,21 +14,21 @@ module "alb" {
 module "helm" {
   source = "./helm"
 
-  aws_region       = var.aws_region
-  aws_workspace    = var.aws_workspace
-  cluster_name     = var.cluster_name
-  docker_username  = var.docker_username
-  docker_password  = var.docker_password
-  docker_email     = var.docker_email
-  helm_values      = local.helm_values
-  microservices    = local.microservices
-  monitors         = local.monitors
-  public_monitors  = local.public_monitors
-  monitors_enabled = var.monitors_enabled
-  monitor_version  = local.monitor_version
-  ingress_scheme   = var.ingress_scheme
-
   acm_certificate_arn = module.alb.acm_certificate_arn
+  aws_region          = var.aws_region
+  aws_workspace       = var.aws_workspace
+  cluster_name        = var.cluster_name
+  docker_email        = var.docker_email
+  docker_password     = var.docker_password
+  docker_username     = var.docker_username
+  helm_values         = local.helm_values
+  ingress_scheme      = var.ingress_scheme
+  logs_bucket         = var.logs_bucket
+  microservices       = local.microservices
+  monitor_version     = local.monitor_version
+  monitors            = local.monitors
+  monitors_enabled    = var.monitors_enabled
+  public_monitors     = local.public_monitors
```

Comment:
[nit] I would move this back up to keep local + top level variables separate from module variables for readability